src/e2e-test/features/aribasource/AribaRunTime.feature

```diff
@@ -0,0 +1,56 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+
+@AribaSource
+@Smoke
+@Regression
+Feature: Ariba Source - Run time scenarios
+
+  @BATCH-TS-ARIBA-DSGN-RNTM-01 @BQ_SINK @FILE_PATH @BQ_SINK_CLEANUP @BQ_TEMP_CLEANUP
+  Scenario: Verify data is transferred correctly to BQ for System Type prod
+    When Open Datafusion Project to configure pipeline
+    And Select plugin: "SAP Ariba" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "SAP Ariba"
+    And Enter input plugin property: "referenceName" with value: "Referencename"
+    And Enter input plugin property: "baseURL" with value: "admin.apiendpoint" for Credentials and Authorization related fields
+    And Enter input plugin property: "realm" with value: "admin.realm" for Credentials and Authorization related fields
+    And Select radio button plugin property: "systemType" with value: "prod"
+    And Enter input plugin property: "viewTemplateName" with value: "SourcingProjectFactSystemView"
+    And Enter input plugin property: "clientId" with value: "admin.clientid" for Credentials and Authorization related fields
+    And Enter input plugin property: "clientSecret" with value: "admin.clientsecret" for Credentials and Authorization related fields
+    And Enter input plugin property: "apiKey" with value: "admin.apiKey" for Credentials and Authorization related fields
+    And Wait with a timeout of 60000 seconds
+    And Validate "SAP Ariba" plugin properties
+    And Capture the generated Output Schema
+    And Close the Plugin Properties page
+    And Select Sink plugin: "BigQueryTable" from the plugins list
+    And Navigate to the properties page of plugin: "BigQuery"
+    And Enter input plugin property: "referenceName" with value: "Reference"
+    And Replace input plugin property: "project" with value: "projectId"
+    And Enter input plugin property: "dataset" with value: "dataset"
+    And Enter input plugin property: "datasetProject" with value: "datasetprojectId"
+    And Enter input plugin property: "table" with value: "bqtarget.table"
+    And Validate "BigQuery" plugin properties
+    And Close the Plugin Properties page
+    And Connect source as "SAP-Ariba" and sink as "BigQueryTable" to establish connection
+    And Wait with a timeout of 60000 seconds
+    And Save and Deploy Pipeline
+    And Run the Pipeline in Runtime
+    And Wait with a timeout of 960000 seconds
+    And Open and capture logs
+    And Verify the pipeline status is "Succeeded"
+    And Close the pipeline logs
+    Then Validate record created in Sink application for ViewTemplate is equal to expected output file "expectedOutputFile"
+    And Wait with a timeout of 60000 seconds
```

Comment:
Why do we need to wait at the end of the testcase? Also, why are sleeping for such a long time? How long does this test take to run?

Reply:
There is a second and minute limit to hit the API so one's the test finishes it hardly take 1 min to configure the next test and validate the plugin, so sometimes it shows null error if the minute limit is exceeded.

Reply:
<img width="622" alt="Screen Shot 2022-10-19 at 12 13 03 PM" src="https://user-images.githubusercontent.com/98015926/196616295-bea2c92c-503e-4ace-a4f7-dd3d54f321c7.png">